src/components/common/FileUploadPdf.vue

```diff
@@ -205,15 +205,15 @@ export default class FileUploadPdf extends Vue {
     try {
       const arrayBuffer = await file.arrayBuffer()
       const data = new Uint8Array(arrayBuffer) // put it in a Uint8Array
-      const document = await this.pdfjsLib.getDocument({ data: data })
```

Comment:
Is it possible we could write some unit tests that can identify any issues? or copy over unit tests from other projects so this doesn't bust again somehow?

Reply:
I'll have a look Travis!

Reply:
Thank you, I think we use this in 3 projects now? it would be awesome if we had some unit tests we could spread across all the repos

Reply:
Yes Travis. However, PDF upload is done differently in Filings UI than the other 2 UIs. The other 2 have a mixin and the actual component while here it's just one file that has everything.

Create UI has some tests there for PDF upload while Edit UI and this here doesn't have many. So, I totally agree with you.

Reply:
Travis, I added some unit tests for the component (I was able to reuse some of the stuff in Create UI). Please see latest commit!